src/containers/org/members/RolesSection.js

```diff
@@ -60,26 +68,24 @@ const RolesSection = ({
           )
         }
       </RolesSectionHeader>
+      <SearchInput
+          onChange={(event :SyntheticEvent<HTMLInputElement>) => debounceSetSearchTerm(event.currentTarget.value)}
+          placeholder="Filter roles" />
       <Checkbox
           checked={selectedRoleId === null}
           label="All Members"
           mode="button"
           onChange={handleOnChangeRoleCheckBox} />
       <Divider margin={0} />
+      <FilteredRoles
+          filterTerm={filterTerm}
+          onChange={handleOnChangeRoleCheckBox}
+          roles={organization.roles}
+          selectedRoleId={selectedRoleId} />
       {
-        organization.roles.map((role :Role) => (
-          <Checkbox
-              checked={role.id === selectedRoleId}
-              data-role-id={role.id}
-              key={role.id}
-              label={role.title}
-              mode="button"
-              onChange={handleOnChangeRoleCheckBox} />
-        ))
-      }
-      {
-        isOwner && isVisibleAddRoleToOrgModal && (
+        isOwner && (
           <AddRoleToOrgModal
+              isVisible={isVisibleAddRoleToOrgModal}
```

Comment:
Why are you making this change?

Reply:
To fix the transition. The modal should always be mounted.